src/chatting/InputChat.tsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+import { useRecoilState } from 'recoil';
+import { userNumber, chatList, selectedUser } from '../atom/atom';
+import { ChatInfo } from '../interface/interface';
+
+const InputChat = () => {
+  const [text, setText] = useState<string>('');
+  const [addUserNum, setAddUserNum] = useRecoilState(userNumber); // 비교하려는 usernum
+  const [newChatList, setNewChatList] = useRecoilState(chatList);
+  const [selectedId, setSelectedId] = useRecoilState(selectedUser); //비교하려는 selectedId >> userID랑 비교
+
+  if (selectedId !== addUserNum) {
+    setSelectedId(1);
+  }
+
+  const submitText = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    console.log('submit?');
+    if (text.trim()) {
+      let newChat: ChatInfo = {
+        userNum: selectedId,
+        addText: text,
+      };
+      setText('');
+      setNewChatList([...newChatList, newChat]);
+    }
+  };
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    console.log('change');
+    setText(e.target.value);
+  };
+
+  return (
+    <InputText onSubmit={() => submitText}>
```

Comment:
이 부분은 Event 객체가 전달되지 않기 때문에, 화살표 함수를 새로 만들지 않고 `submitText` 함수를 그대로 전달해야 합니다! 또는 두번째 코드와 같이 작성해야 할 것 같습니다~
```suggestion
    <InputText onSubmit={submitText}>
```
```suggestion
    <InputText onSubmit={(e)=>submitText(e)}>
```

Reply:
네 맞습니다ㅠㅠ 제가 수정하지 못 하고 푸쉬한 채로 제출해버렸네요😿